decentraland/unity-renderer
# feat: biw custom asset pack

## What this PR includes?

This PR includes 

- Pink cube when you has deleted your custom asset pack to show that the asset is no longer available
- A new message when you are trying to publish with error entities
- Unit test


## How to test it?

1. Go to https://builder.decentraland.io/ 
2. Create a custom asset pack
3. Go to https://play.decentraland.zone/index.html?ENABLE_BUILDER_IN_WORLD&ENV=zone&realm=artemis-amber&renderer=urn%3Adecentraland%3Aoff-chain%3Arenderer-artifacts%3Afeat%2Fbiw-custom-asset-pack
4. Edit a land in builder in world
5. Add an asset from the custom asset pack
6. Delete the asset pack in builder.decentraland.io
7. Reload the decent land explorer 
8. Edit the same land in builder in world and you will see the pink cubes

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
